roles/maas_postgres/tasks/o11y_pg.yaml

```diff
@@ -0,0 +1,60 @@
+---
+- name: "Create postgres-exporter directory"
+  ansible.builtin.file:
+    path: "{{ o11y_postgres_exporter_dir }}"
+    state: "{{ 'directory' if (o11y_enable and 'maas_postgres' in group_names) else 'absent' }}"
+    mode: 0755
+  changed_when: false
+
+- name: "Discover host architecture"
```

Comment:
move this to `site.yaml` (all hosts) and call this fact `ubuntu_arch` (or something similar). We are going to need this info more times